<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ServoParamAllWriteTipWidget</class>
 <widget class="QWidget" name="ServoParamAllWriteTipWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>250</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>350</width>
    <height>250</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget#ServoParamAllWriteTipWidget{
    border: 1px solid #8f8f91;
    border-radius:6px;
    background-color: rgb(255, 255, 255);
}</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="labelMessage">
       <property name="text">
        <string>覆盖伺服参数需保证机器下使能。
升级成功后首次运行机器时，
请远离机械臂确保自身安全。</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter|Qt::AlignVCenter">
      <widget class="QLabel" name="labelDoing">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item alignment="Qt::AlignHCenter">
      <widget class="QPushButton" name="btnClose">
       <property name="minimumSize">
        <size>
         <width>88</width>
         <height>44</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QPushButton
{
    background-color: #0047BB;
    color: rgb(255, 255, 255);
    border:none;
    padding-left:24px;
    padding-right:24px;
	border-radius: 4px;
}
</string>
       </property>
       <property name="text">
        <string>关闭</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="btnCancel">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">  background-color: #C7C7C7;
  color: rgb(255, 255, 255);
  border:none;
  padding-left:24px;
padding-right:24px;
border-radius: 4px;</string>
         </property>
         <property name="text">
          <string>取消</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnConfirm">
         <property name="minimumSize">
          <size>
           <width>88</width>
           <height>44</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>88</width>
           <height>44</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">QPushButton#btnConfirm
{
    background-color: #0047BB;
    color: rgb(255, 255, 255);
    border:none;
    padding-left:24px;
    padding-right:24px;
	border-radius: 4px;
}
</string>
         </property>
         <property name="text">
          <string>确认</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
